cmd/opm/index/delete.go

```diff
@@ -75,15 +76,11 @@ func runIndexDeleteCmdFunc(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	containerTool, err := cmd.Flags().GetString("container-tool")
+	pullTool, buildTool, err := getContainerTools(cmd)
 	if err != nil {
 		return err
 	}
 
-	if containerTool == "none" {
```

Comment:
I know we have a lot of conversations around non-tool. I couldn't keep track of what's going on there, Is there any reason we are removing the check here? Thanks.

otherwise lgtm

Reply:
The concept of that check was just moved into the existing private method `getContainerTools()`, which I'm reusing the logic for:

https://github.com/operator-framework/operator-registry/blob/master/cmd/opm/index/add.go#L162

Reply:
ahh, nice. Thanks

/lgtm
/hold 
waiting for a second review